tests/Behat/Page/Admin/Product/ShowPageInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\PriceHistoryPlugin\Behat\Page\Admin\Product;
+
+use Sylius\Behat\Page\Admin\Product\ShowPageInterface as BaseShowPageInterface;
+
+interface ShowPageInterface extends BaseShowPageInterface
+{
+    public function openChannelPricingLogEntryIndexPageForSimpleProduct(string $channelName): void;
```

Comment:
```suggestion
    public function showPriceHistoryInChannel(string $channelName): void;
```
?

Reply:
refactored it slightly different with new PricingElement wdyt?